src/cmds/hardware/lsblk/lsblk.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file
+ *
+ * @date Dec 4, 2019
+ * @date Anton Bondarev
+ */
+#include <stdio.h>
+#include <inttypes.h>
+
+#include <drivers/block_dev.h>
+
+int main(int argc, char **argv) {
+	int i;
+	struct block_dev **bdevs;
+
+	bdevs = get_bdev_tab();
+	assert(bdevs);
+
+	printf("Block devices:\n");
+	printf("  ID |  NAME          |    SIZE    | TYPE\n");
+
+	for (i = 0; i < MAX_BDEV_QUANTITY; i++) {
+		if (bdevs[i] && !block_dev_get_parent(bdevs[i])) {
```

Comment:
Why don't use `bdevs[i]->parent_bdev` directly? We access other structure fields here anyway (`id` and `name`).